corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -212,7 +212,7 @@ <h2>{% trans "Manage Organization Structure" %}</h2>
   </div>
 {% endif %}
 
-<div class="col-lg-12">
+<div class="col-lg-12" style="clear: both">
```

Comment:
I think the canonical bootstrap way to handle this is to wrap this div in a div with `class='row'`. If that doesn't work, using the `clearfix` class is a tiny bit better than inline styling.